content/en/security/application_security/threats/add-user-info.md

```diff
@@ -623,7 +623,10 @@ The following examples show how to track login events or custom events (using si
 
 {{< /programming-lang >}}
 {{< programming-lang lang="nodejs" >}}
-Starting in dd-trace-js v3.13.1, you can use the Node.js tracer's API to track user events.
+
+Starting in dd-trace-js v3.13.1, you can use the Node.js tracer API to track user events. In dd-trace-js v5.48.0, new methods were introduced under the `eventTrackingV2` namespace, while the existing ones were retained for compatibility.
+
+Note that the legacy methods `trackUserLoginSuccessEvent` and `trackUserLoginFailureEvent` are now deprecated in favor of the new methods `eventTrackingV2.trackUserLoginSuccess` and `eventTrackingV2.trackUserLoginFailure`, respectively.
```

Comment:
I'd suggest t moving this in the migration section. New users don't necessarily care about details about the legacy SDK